Remove "v"-Prefix for $ALPINE_BRANCH Environment Variable

robin-rpr opened the pull request:
This Pull Request allows usage of Alpine Branches like "edge".

Should this Pull Request be approved I'll be happy to create one on the Original (non-mirror) of this Repository.

Cheers!

bboehmke commented:
Looks like a good change.

It would be perfect to also support the old version branch without the `v` but I currently dont have an idea how this can be achieved in an proper way.

Maybe you have an idea?

robin-rpr commented:
Thanks @bboehmke !

I have added support for this with 

`ALPINE_BRANCH=$(echo $ALPINE_BRANCH | sed '/^[^v].*[0-9]/s/^/v/')`

It allows usage of all Branch ([current](https://alpinelinux.org/releases/)) names, but appends missing `v` to Semver'ish strings.

Cheers!

PS: Also updated deprecated link in README.md

bboehmke commented on .github/workflows/master.yml:
```diff
@@ -0,0 +1,21 @@
+          name: ${{ secrets.REGISTRY_USERNAME }}/raspi-alpine-builder/raspi-alpine-builder
+          username: ${{ secrets.REGISTRY_USERNAME }}
+          password: ${{ secrets.REGISTRY_PASSWORD }}
+          registry: ${{ secrets.REGISTRY_HOST }}
```
I am fine with adding some configuration for github actions but I think it would be better in a separate merge request.

robin-rpr replied:
Yup! This wasn't supposed to be a part of this Commit. I'll make sure to exclude it =)

bboehmke commented on resources/build.sh:
```diff
@@ -24,6 +24,7 @@ set -e
 : ${INPUT_PATH:="/input"}
 : ${CUSTOM_IMAGE_SCRIPT:="image.sh"}
 
+ALPINE_BRANCH=$(echo $ALPINE_BRANCH | sed '/^[^v].*[0-9]/s/^/v/')
```
Nice idea. 

But I am not sure if I understand all cases covered by the regex.
I would say a simply "add a `v` if the version starts with a number" would be enough.

Something like:
```suggestion
ALPINE_BRANCH=$(echo $ALPINE_BRANCH | sed '/^[0-9]/s/^/v/')
```

robin-rpr replied:
LGTM!

bboehmke reviewed (commented):
Because there are a lot of commits for this (small) change, I will probably squash all changes into one commit before merging them into master.

If you prefer you can also cleanup the commit yourself.

robin-rpr commented:
Feel free to Squash the Pull Request =) For now everything looks good to me!
